tests/integration/services/activation/engine/test_podman.py

```diff
@@ -262,16 +263,32 @@ def test_engine_start_with_none_image_url(
         engine.start(request, log_handler)
 
 
+@pytest.mark.parametrize(
+    "image_url",
+    [
+        "quay.io/ansible/ansible-rulebook",
+        "quay.io:5004/ansible/ansible-rulebook",
+        "quay.io:5004/ansible/ansible-rulebook:main",
+        "quay.io/ansible/ansible-rulebook@sha256:756dd6a40b62975ea10d83a577ed6b1cddd545de17d9fb56b3d4e80e166826df",  # noqa: E501
+        "quay.io:5004/ansible/ansible-rulebook@sha256:756dd6a40b62975ea10d83a577ed6b1cddd545de17d9fb56b3d4e80e166826df",  # noqa: E501
+        "quay.io:443/ansible/ansible-rulebook@sha512:756dd6a40b62975ea10d83a577ed6b1cddd545de17d9fb56b3d4e80e166826df",  # noqa: E501
+    ],
```

Comment:
can you add a testcase like the mentioned one by @Zokormazo ?
`http://quay.io/fedora/fedora:42@sha256:68779a51b655f6bfeef08429fe31c33508f6a0d6beaf5660de094d49948a8e37`

Reply:
added.